day22/permutations.py

```diff
@@ -0,0 +1,20 @@
+def permutations(arr):
+    allPermutations = list()
+
+    if len(arr) <= 1:
```

Comment:
added benefit is that you can hard code the base case return value to be `[[]]` which imo looks better as a base case